ProcessMaker/Repositories/CaseSyncRepository.php

```diff
@@ -113,13 +135,32 @@ private static function processTokens($instance, &$cpData, &$csRequestTokens, &$
     {
         foreach ($instance->tokens as $token) {
             if (in_array($token->element_type, CaseUtils::ALLOWED_REQUEST_TOKENS)) {
-                $cpData['processes'] = CaseUtils::storeProcesses($instance, $cpData['processes']);
-                $cpData['requests'] = CaseUtils::storeRequests($instance, $cpData['requests']);
-                $cpData['request_tokens'] = CaseUtils::storeRequestTokens($token->getKey(), $cpData['request_tokens']);
-                $cpData['tasks'] = CaseUtils::storeTasks($token, $cpData['tasks']);
-
-                $csRequestTokens = CaseUtils::storeRequestTokens($token->getKey(), $csRequestTokens);
-                $csTasks = CaseUtils::storeTasks($token, $csTasks);
+                $processData = [
+                    'id' => $instance->process->id,
+                    'name' => $instance->process->name,
+                ];
+
+                $requestData = [
+                    'id' => $instance->id,
+                    'name' => $instance->name,
+                    'parent_request_id' => $instance?->parentRequest?->id,
+                ];
+
+                $taskData = [
+                    'id' => $token->getKey(),
+                    'element_id' => $token->element_id,
+                    'name' => $token->element_name,
+                    'process_id' => $token->process_id,
+                    'element_type' => $token->element_type,
+                ];
+
+                $cpData['processes'] = CaseUtils::storeProcesses($cpData['processes'], $processData);
```

Comment:
improve $cpData variable names to better understanding

Reply:
It was fixed